locales/zh_CN/LC_MESSAGES/hello.po

```diff
@@ -213,125 +210,117 @@ msgstr "（可用: 该系统上有最完整的支持；不可用: 由于平台
 
 #: ../../hello.rst:91
 msgid ""
-"With ``arch=ti.gpu``, Taichi will first try to run with CUDA. If CUDA is "
-"not supported on your machine, Taichi will fall back on Metal or OpenGL. "
-"If no GPU backend (CUDA, Metal, or OpenGL) is supported, Taichi will fall"
-" back on CPUs."
+"With ``arch=ti.gpu``, Taichi will first try to run with CUDA. If CUDA is not supported on your "
+"machine, Taichi will fall back on Metal or OpenGL. If no GPU backend (CUDA, Metal, or OpenGL) is "
+"supported, Taichi will fall back on CPUs."
 msgstr ""
+"在参数 ``arch=ti.gpu`` 下，Taichi 将首先尝试在 CUDA 上运行。如果你的设备不支持 CUDA，那么 Taichi 将会"
+"降级到 Metal 或 OpenGL。如果不支持 GPU 后端（CUDA、Metal 或 OpenGL），Taichi 将依赖于 CPU 运行。"
```

Comment:
```suggestion
"尝试 Metal 或 OpenGL。如果所在平台不支持 GPU 后端（CUDA、Metal 或 OpenGL），Taichi 将默认在 CPU 运行。"
```

This is probably not a word-for-word translation, but maybe it reads more fluently in Chinese..?

Reply:
We used the word ``尝试`` twice.This might a little odd when we reading
How about this
> Taichi 将首先尝试在 CUDA 上运行。如果你的设备不支持 CUDA，那么 Taichi 将会转到 Metal 或 OpenGL。如果所在平台不支持 GPU 后端（CUDA、Metal 或 OpenGL），Taichi 将默认在 CPU 运行。

How do you like it?